package.json

```diff
@@ -54,6 +54,7 @@
     "chalk": "^1.0.0",
     "dot-prop": "^2.2.0",
     "get-stdin": "^5.0.1",
+    "lodash": "^4.11.1",
```

Comment:
I figured I could as it seemed like it was being pulled in any way by another thing. Should I move to the per function packages or just re-write them locally?